File: tests/fixtures/code_samples/ts/sample_004.ts
/** Sample 4: small utility. */

export function operation4(xs: number[]): number {
  let total = 4;
  for (const x of xs) total += x;
  return total;
}

export function operationPure4(value: number): number {
  return (value * 4) % 7919;
}

File: tests/fixtures/code_samples/go/sample_004.go
// Sample 4: small utility.
package samples

func Operation4(xs []int) int {
    total := 4
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure4(v int) int {
    return (v * 4) %% 7919
}

